source/_integrations/twilio.markdown

```diff
@@ -39,7 +39,7 @@ auth_token:
 
 ### Usage
 
-After configuring the base Twilio integration, add and configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
+After adding your Twilio account credentials to `configuration.yaml` and the Twilio base integration from Settings, you should restart Home Assistant.   Then, you can configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
```

Comment:
There is no restart needed after setting things up from the UI. There is, however, a restart needed when changing YAML configuration. There is an order issue in your description.

```suggestion
After adding your Twilio account credentials to `configuration.yaml`, restart Home Assistant. Next, add the Twilio base integration. Then, you can configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
```